packages/icons/icons.json

```
@@ -1969,182 +1969,182 @@
         "group": "table",
         "codepoint": 62219
     },
     {
         "displayName": "Manual",
         "iconName": "manual",
-        "tags": "guide, instruction",
+        "tags": "guide, instruction, documentation, docs, book, open",
         "group": "interface",
         "codepoint": 62005
     },
     {
         "displayName": "Add row: top",
         "iconName": "add-row-top",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add, add row",
         "group": "table",
         "codepoint": 61702
     },
     {
         "displayName": "Add row: bottom",
         "iconName": "add-row-bottom",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, widget, add",
         "group": "table",
         "codepoint": 61701
     },
     {
         "displayName": "Add column: left",
         "iconName": "add-column-left",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add tab, add tab left, add window, add column, add right",
         "group": "table",
         "codepoint": 61698
     },
     {
         "displayName": "Add column: right",
         "iconName": "add-column-right",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add section right, add, new, medicine",
         "group": "table",
         "codepoint": 61699
     },
     {
         "displayName": "Remove row: top",
         "iconName": "remove-row-top",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, mail, cabinet, files, storage box",
         "group": "table",
         "codepoint": 62095
     },
     {
         "displayName": "Remove row: bottom",
         "iconName": "remove-row-bottom",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, box, files, dashboard, application, container, oven",
         "group": "table",
         "codepoint": 62094
     },
     {
         "displayName": "Remove column: left",
         "iconName": "remove-column-left",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, door, open, box, application, sidebar, dashboard",
         "group": "table",
         "codepoint": 62091
     },
     {
         "displayName": "Remove column: right",
         "iconName": "remove-column-right",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, module, section, widget, remove panel, remove widget, divide panel, library, book, collection, drawer, sidebar, menu",
         "group": "table",
         "codepoint": 62092
     },
     {
         "displayName": "Double chevron: left",
         "iconName": "double-chevron-left",
-        "tags": "arrows, multiple, direction",
+        "tags": "arrows, multiple, direction, chevrons left, expand left, shift left, double left chevron, go left, inbox, new email, skip backward, quick reverse, move left, left arrows, open bracket, back",
```

```suggestion
        "tags": "arrows, multiple, direction, expand, shift, go left, skip backward, quick reverse, move, open bracket, back",
```

not sure what this has to do with emails